content/tutorials/esc-external-secret-operator/index.md

```diff
@@ -0,0 +1,491 @@
+---
+title: "Integrate with External Secrets Operator (ESO)"
+title_tag: Integrate with External Secrets Operator (ESO) | Learn Pulumi
+h1: "Integrate with External Secrets Operator (ESO)"
+layout: single
+description: |
+  Pulumi ESC integrates with the External Secrets Operator (ESO) to manage and deliver secrets in Kubernetes clusters.
+meta_desc: Pulumi ESC integrates with the External Secrets Operator (ESO) to manage and deliver secrets in Kubernetes clusters.
+weight: 999
+meta_image: meta.png
+summary: |
+  In this tutorial, you will learn how to deploy the External Secrets Operator and use it to manage secrets stored in Pulumi ESC.
+youll_learn:
+- How to deploy the External Secrets Operator using Helm or Pulumi
+- How to manage secrets stored in Pulumi ESC using External Secrets Operator
+- How to push secrets from Kubernetes to Pulumi ESC
+estimated_time: 15 minutes
+collections:
+- pulumi-esc
+- kubernetes
+collections_weight: 2
+---
+
+## Overview
+
+[External Secrets Operator](https://external-secrets.io/latest/) is a Kubernetes operator that integrates external secret management systems with Kubernetes. By using External Secrets Operator, you have several advantages over Kubernetes native secrets:
+
+- Sensitive data is stored and managed by an external service outside the Kubernetes cluster and this leads to better security and compliance.
+- External Secrets Operator supports multiple different sources, so you can use the same operator to manage secrets and configuration from different sources.
+- Take advantage of the advanced features of the secret provider, such as encryption of data at rest and scenarios like secret rotation.
+
+Since version `0.10.0` External Secrets Operator supports Pulumi ESC as a secret provider.
+
+## Prerequisites
```

Comment:
This should actually be defined in the frontmatter to align with the general format and structure of tutorials.

```
# A list of tutorial prerequisites. Markdown is fine. Keep it simple; no need to be exhaustive here.
prereqs:
    - The [Pulumi CLI](/docs/install/)
    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
    - An [Azure account](https://azure.microsoft.com/en-us/pricing/purchase-options/azure-account)
    - The [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
    - Your desired [language runtime installed](/docs/iac/get-started/azure/begin/#install-language-runtime)
```

Reply:
I have also been putting the content of my "Overview" in the summary section of the frontmatter.